pages/index.js

```diff
@@ -1,14 +1,29 @@
 import Head from 'next/head'
 import Footer from '../components/Footer'
 import Navbar from '../components/Navbar'
```

Comment:
Imported but never used on this page. Nothing major and might not hit performance